std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/LongStorage.java

```diff
@@ -198,4 +187,70 @@ public LongStorage widen(IntegerType widerType) {
     assert widerType.fits(getType());
     return new LongStorage(data, (int) getSize(), getIsNothingMap(), widerType);
   }
+
+  /** Allow access to the underlying data array for copying. */
+  public long[] getArray() {
+    return data;
+  }
+
+  @Override
+  public ColumnLongStorageIterator iterator() {
```

Comment:
Why don't se use `java.util.Iterator<Long>`? Why don't we make the storage `implements Iterable<Long>` so one can write `for (var l : storage)`?

Reply:
It does and we can.